cmd/olm/cleanup.go

```diff
@@ -239,3 +259,33 @@ func crossNamespaceOwnerReferenceRemoval(kind string, uidNamespaces map[types.UI
 		return
 	}
 }
+
+// ensureAPIServiceLabels checks the labels of existing APIService objects to ensure ownership is set correctly
+// If the APIService label does not correspond to the expected packageserver owner the APIService labels are updated
+func ensureAPIServiceLabels(c clientset.Interface) error {
+	apiService, err := c.ApiregistrationV1().APIServices().Get(packageserverName, metav1.GetOptions{})
+	if err != nil && !k8serrors.IsNotFound(err) {
+		return err
+	}
+	if apiService == nil {
```

Comment:
My thought process here is to assume this is running on both OCP and vanilla kube. Since running the packageserver and APIService is not a requirement, if there is no APIService it should not be an error. The user could only be using parts of OLM. Does that sound right?

Reply:
Yep, that seems correct.